account_asset_management/views/account_invoice_line.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_invoice_line_form" model="ir.ui.view">
```

Comment:
If I remember correctly everything should be 4 spaces.

Reply:
I had a look into the OCA guidelines and didn't find any recommendation on xml file indent spaces, hence let's keep it as is for the moment and refine first the OCA recommendation before we make such changes.